src/SMuFF.cpp

```diff
@@ -385,7 +384,7 @@ void setup() {
   setupHBridge();
   getStoredData();                    // read EEPROM.DAT from SD-Card; this call must happen after setupSteppers()
   //__debug(PSTR("readSequences start"));
-  uint16_t now = millis();
```

Comment:
On STM32 millis() returns a uint32_t. At least that's what the documentation says.

Reply:
It was uint16_t and now it's uint32_t